packages/server/docker-compose.yml

```diff
@@ -1,14 +1,27 @@
 version: '3.4'
 
 volumes:
-  db-data:
+  pioneer-db-data:
+
+networks:
+  default:
+    name: joystream_default
+    external: true
 
 services:
-  db:
+  pioneer-api:
+    image: thesan/pioneer-backend # TODO change this to joystream/...
+    build:
+      context: ../..
+      dockerfile: backend.Dockerfile
+    ports: ['${PORT}:${PORT}']
```

Comment:
Haven't seen this syntax used in docker-compose yet, always the regular list
```
ports:
  - '${PORT};${PORT}'
```
Super small, don't bother to change but I find the above syntax easier to parse automatically :D

Reply:
Yeah I tend to favor terser syntax to a fault !